package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "homebridge-ecobee3-sensors",
-  "version": "0.1.11",
```

Comment:
I usually bump up the version number myself when I do a release and publish to NPM.